extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java

```diff
@@ -10,11 +10,7 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
-import org.jboss.jandex.AnnotationInstance;
-import org.jboss.jandex.AnnotationTarget;
-import org.jboss.jandex.ClassInfo;
-import org.jboss.jandex.DotName;
-import org.jboss.jandex.Type;
+import org.jboss.jandex.*;
```

Comment:
Avoid star imports

Reply:
If I'm not mistaken, that's the work of the formatter, I didn't do that explicitly.

Reply:
Indeed, my IDE was not properly configured… Thought that running `mvn -Dquickly` was formatting the code as well.